thingif/src/main/java/com/kii/thingif/temporal/base/BaseAnd.java

```diff
@@ -0,0 +1,8 @@
+package com.kii.thingif.temporal.base;
+
+import java.util.List;
+
+public interface BaseAnd<T> {
```

Comment:
I expect `AndClauseInQuery` and `AndClauseInTrigger` solve the problem. I think developer should not directly use `BaseAnd`.